captain/internal/manager.py

```diff
@@ -19,7 +21,25 @@ def __init__(self):
 
 # Manager for Test Sequencer activities
 class TSManager(WSManager):
-    pass  # could potentially add more things in the future for this, that's why we keep it as such
+
+    def __init__(self):
+        self.runner: asyncio.Runner | None = None  # holds the running sequencer
```

Comment:
it's better that the "runner" is not a asyncio runner by rather a thread. The reason being if we want to cancel a long test atom, it will wait for it to finish first. (Do we leave it like this for now?)

Reply:
Currently, the test sequencer is run as an async function (before having the cancel functionnality) I'm just providing a way to cut the task between tests. A test won't be kill because as Izi pointed out, it's run in a subprocess and the exception raised by stopping the async "queue" won't kill it. 
### Upside:
- It doesn't stop a test, so if a test need a cleanup from pytest, this one will be run.
- Same behavior as the Flowchart
### Down side:
- Won't kill a long running test (Or even a infinit loop)